Docs/Luadoc/LuaDocumentation.xml

```diff
@@ -3598,6 +3602,9 @@ NETWORK:HttpRequest{
 	errorMessage="access to https://api.example.com is not allowed",
 }
 </code></pre>
+	If the <code>downloadFile</code> parameter is set, the response body is not accumulated in memory, but written to <code>/Downloads/${downloadFile}</code>.
+	The file is available in the <code>onResponse</code> callback where it can be copied to another location.
+	It is removed after the callback completed.

```

Comment:
`The file is deleted once the callback returns.`

Reply:
fixed